azure-pipelines.yml

```diff
@@ -14,7 +14,7 @@ jobs:
            docker pull i386/ubuntu:bionic
            docker run -v $(pwd):/numpy i386/ubuntu:bionic /bin/bash -c "cd numpy && \
            apt-get -y update && \
-           apt-get -y install python3.6-dev python3-pip locales && \
+           apt-get -y install python3.6-dev python3-venv python3-pip locales && \
```

Comment:
No longer need `python3-venv`?

Reply:
True, but on the other hand it couldn't hurt and makes this environment consistent with the other linux ones one travis

Reply:
Come, come, you could install lots of things that don't hurt, but that is not a reason to do it.

Reply:
ok, removing